sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2803,6 +2854,49 @@ def _provision(
             if dryrun:
                 record = global_user_state.get_cluster_from_name(cluster_name)
                 return record['handle'] if record is not None else None
+
+            # We pass handle in the config dict for the new provisioner
```

Comment:
This is not needed as we pass handle to config dict for non new provisioner case as well.
```suggestion
```